plugins/opa/examples/rollout-strategy/policy.rego

```diff
@@ -0,0 +1,30 @@
+package fairwinds
+
+blockedNamespace(elem) {
+  ns := elem.parameters.blocklist[_]
+  elem.metadata.namespace == ns
+}
+
+# If this block evaluates to true an action item created
+rollingUpdate[actionItem] {
+
+  # Checks to see if this resource should be skipped or not based on blocked namespaces.
+  not blockedNamespace(input)
+  
+  # These lines check to see if the maxUnavailable field is a percentage, and if so to strip it and converts it to a number for evaluation.
+  maxUnavailable := input.spec.strategy.rollingUpdate.maxUnavailable
+  contains(maxUnavailable, "%")
+  maxUnavailble = replace(maxUnavailable, "%", "")
+  newMaxUnavailable = to_number(maxUnavailble)
+  
+  # If newMaxUnavailable is greater than 25 then create an action item.
+  newMaxUnavailable = 25
```

Comment:
would suggest making this `==` per [these docs](https://www.openpolicyagent.org/docs/latest/policy-language/#equality-assignment-comparison-and-unification)

Actually I'm not sure if `=` works here? I don't really understand how `=` works in rego.

Reply:
Oh actually - reading your comment - I think this should be `newMaxUnavailable > 25`

Reply:
Ooh, good catch on newMaxUnavailable > 25! Not sure how that ended up as an =. That should now be fixed.

The single equals sign works in my test cluster, and when I add == for line 17 and 18 I get the following errors:
fairwinds:17: rego_unsafe_var_error: var maxUnavailble is unsafe
fairwinds:18: rego_unsafe_var_error: var newMaxUnavailable is unsafe